server/routes/message/view-group-message.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const { HttpError } = require('../../utils/errors');
+const { Message } = require('../../models');
+const { authenticateToken } = require('../../utils/token-helpers');
+
+const router = express.Router();
+router.get('/:id/messages', authenticateToken, async (req, res) => {
+    const { id } = req.user;
+    const groupId  = Number(req.params.id);
+
+    try {
+        const message = new Message(id,groupId);
```

Comment:
I  don't think it's a good design to create a dummy instance of Message so that we can fetch group messages. 

The method to get group messages should be a static method on the Message class and it should be used as shown below. 

```suggestion
        const messages = Message.fetchGroupMessages(id,groupId);
```

Reply:
It cannot be a static method because it uses `this` keyword in its internal implementation

Reply:
You can refactor the internal implementation not to use this keyword and have the dependencies passed down as variables